src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -188,7 +191,9 @@ void CallstackThreadBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assemb
 
 void CallstackThreadBar::OnRelease() {
   CaptureViewElement::OnRelease();
-  SelectCallstacks();
+  if (!absl::GetFlag(FLAGS_time_range_selection)) {
```

Comment:
Please add the same comment here to explain why we are using this anymore (because it's replaced by time range selection)